resources/asciidoctor/spec/copy_images_spec.rb

```diff
@@ -6,7 +6,7 @@
 require 'fileutils'
 require 'tmpdir'
 
-RSpec.describe CopyImages::CopyImages do
+RSpec.describe CopyImages do
```

Comment:
you can also just write `describe` without calling it on `Rspec`

Reply:
That causes an undefined method error for me.

Reply:
hmm, maybe your version of `RSpec` is older
Anyway, it doesn't really matter how you call `describe`